harness/determined/cli/user.py

```diff
@@ -218,6 +243,110 @@ def edit(args: argparse.Namespace) -> None:
         raise errors.CliError("No field provided. Use 'det user edit -h' for usage.")
 
 
+def render_token_info(token_info: Sequence[bindings.v1TokenInfo]) -> None:
+    values = []
+    for ti in token_info:
+        value = [
+            ti.id,
+            ti.userId,
+            ti.description,
+            ti.createdAt,
+            ti.expiry,
+            ti.revoked,
+            ti.tokenType,
+        ]
+        values.append(value)
```

Comment:
note on writing more pythonic code: use a list comprehension here, rather that a for loop with appending.

```python
values = [
  [t.id, t.userId, t.description, t.createdAt, t.expiry, t.revoked, t.tokenType]
  for t in token_info
]
```

Reply:
Resolving it as it is now implemented